public/app/features/commandPalette/CommandPalette.tsx

```diff
@@ -75,25 +75,44 @@ export const CommandPalette = () => {
 };
 
 interface RenderResultsProps {
-  dashboardResults: CommandPaletteAction[];
+  searchResults: CommandPaletteAction[];
 }
 
-const RenderResults = ({ dashboardResults }: RenderResultsProps) => {
-  const { results, rootActionId } = useMatches();
+const RenderResults = ({ searchResults }: RenderResultsProps) => {
+  const { results: kbarResults, rootActionId } = useMatches();
   const styles = useStyles2(getSearchStyles);
   const dashboardsSectionTitle = t('command-palette.section.dashboard-search-results', 'Dashboards');
+  const foldersSectionTitle = t('command-palette.section.folder-search-results', 'Folders');
   // because dashboard search results aren't registered as actions, we need to manually
   // convert them to ActionImpls before passing them as items to KBarResults
   const dashboardResultItems = useMemo(
-    () => dashboardResults.map((dashboard) => new ActionImpl(dashboard, { store: {} })),
-    [dashboardResults]
+    () =>
+      searchResults
+        .filter((item) => item.id.startsWith('go/dashboard'))
```

Comment:
not sure whether it's worth adding an explicit new type property to our `CommandPaletteAction`? using the id feels a little brittle 🤔

Reply:
also could rewrite this into a generic for loop to avoid looping the array twice? lmk what you think